test/linalg/matmul.jl

```diff
@@ -5,189 +5,279 @@ using Base.Test
 ## Test Julia fallbacks to BLAS routines
 
 # matrices with zero dimensions
-for i = 1:10
```

Comment:
blame says @andreasnoack https://github.com/JuliaLang/julia/commit/687d2b0da6c5e8cff895e8c6f7f8e4b6866b7997 - maybe because `Array` would be uninitialized in the non-zero-dimensional case?